package.json

```diff
@@ -15,7 +15,7 @@
     "@types/node": "^6.0.46",
     "@types/q": "^0.0.32",
     "@types/selenium-webdriver": "~2.53.39",
-    "blocking-proxy": "0.0.5",
+    "blocking-proxy": "^0.0.5",
```

Comment:
Yeah, I was hoping to be able to get out fixes without having to publish a new Protractor. But you're right, with things as they are it's probably better to just leave pinned.